backend/src/main/scala/com/softwaremill/adopttapir/template/scala/APIDefinitionsView.scala

```diff
@@ -51,4 +58,32 @@ object APIDefinitionsView {
       )
     )
   }
+
+  private object DocumentationEndpoint {
+
+    def prepareDocEndpoints(projectName: String, serverEffect: ServerEffect): PlainLogicWithImports = {
+      val endpoints: List[String] = List("helloEndpoint")
+
+      PlainLogicWithImports(prepareCode(projectName, serverEffect, endpoints), prepareImports(serverEffect))
```

Comment:
maybe rename `PlainLogicWithImports` to simply `Code` which has `imports` & `body` sections?